sdk/communication/communication-phone-numbers/test/public/siprouting/utils/recordedClient.ts

```diff
@@ -117,3 +119,18 @@ export async function createRecordedClientWithToken(
   // casting is a workaround to enable min-max testing
   return { client, recorder };
 }
+
+export async function clearSipConfiguration(): Promise<void> {
+  const client = new SipRoutingClient(env.COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING ?? "");
+  await client.setRoutes([]);
+  await client.setTrunks([]);
+}
+
+export function getUniqueFqdn(order: string): string {
+  const length = 12;
```

Comment:
Same here. I would use a different implementation that doesn't involve a loop that breaks by a randomizer.

Reply:
Used UUIDs instead.